tuxemon/core/states/combat/combat_animations.py

```diff
@@ -146,10 +146,8 @@ def func():
         self.task(capdev.kill, fall_time + delay + fade_duration)
 
         # load monster and set in final position
-        monster_sprite = self.load_sprite(
-            monster.back_battle_sprite if npc.isplayer else monster.front_battle_sprite,
-            midbottom=feet,
-        )
+        monster_sprite = monster.get_sprite("back" if npc.isplayer else "front",
```

Comment:
notice that the original load_sprite function was 
```python
        layer = kwargs.pop('layer', 0)
        sprite = tools.load_sprite(filename, **kwargs)
        self.sprites.add(sprite, layer=layer)
        return sprite
```
The crucial part is `self.sprites.add(sprite, layer=layer)`.  Sprite groups contain sprites.  if the sprite is not in it; it won't be rendered.

add `self.sprites.add(monster_sprite)`